src/renderer/browser/util/createArrayScript.js

```diff
@@ -0,0 +1,14 @@
+import Neon from '@cityofzion/neon-js';
+
+import encode from 'shared/util/encodeArgs';
+
+const createArrayScript = (scripts, encodeArgs) => {
+  const args = scripts.map((script) => {
+    return encodeArgs ? encode(script) : script;
+  });
+
+  // Create script
+  return Neon.create.script(args);
```

Comment:
I don't think this is right.  I think we want something more like this:

```ruby
import createScript from 'shared/util/createScript';

const createArrayScript = (scripts, encodeArgs) => {
  return scripts.map(createScript);
};
```

That's not quite right either since we need to concatenate the scripts probably (using `sc` import from neon-js?), but the current logic looks like it's disregarding the `scriptHash` and `operation`.

Reply:
No not really, the Neon.create.script needs to be called with an array (it takes both an object or an array of objects), I wrongfully used the entire array instead of just the args of each element.

Reply:
Should be fixed with my new push @mhuggins